data/json/recipes/recipe_electronics.json

```diff
@@ -2239,5 +2239,33 @@
     [ [ "power_supply", 1 ] ],
     [ [ "cable", 5 ] ]
   ]
+},{
+  "type" : "recipe",
+  "result": "vac_sealer",
+  "category": "CC_ELECTRONIC",
+  "subcategory": "CSC_ELECTRONIC_TOOLS",
+  "skill_used": "electronics",
+  "skills_required": [ "mechanics", 2 ],
+  "difficulty": 4,
+  "time": 40000,
+  "reversible": true,
+  "autolearn": false,
+  "book_learn": [[ "advanced_electronics", 4 ] , [ "mag_electronics", 6 ] , [ "manual_electronics", 5 ]],
+  "qualities" : [
+    {"id":"SCREW","level":1,"amount":1}
+  ], "components": [
+    [
+      [ "scrap", 4 ]
+    ],
+    [
+      [ "element", 2 ]
+    ],
+    [
+      [ "hose", 1 ]
+    ],
+    [
+      [ "power_supply", 1 ]
```

Comment:
I'd like to push back on the ratcheting up of what is craftable.
Everything is not supposed to be craftable.  At the very least this needs
an uncraftable 'air pump' part that can be harvested somewhere, but
building gaskets/pistons etc out of scrap is a bit much.